ui/widgets/menu.js

```diff
@@ -236,9 +236,12 @@ return $.widget( "ui.menu", {
 		default:
 			preventDefault = false;
 			prev = this.previousFilter || "";
-			character = String.fromCharCode( event.keyCode );
 			skip = false;
 
+			// Support number pad values
+			character = event.keyCode >= 96 && event.keyCode <= 105 ?
+				event.keyCode - 96 : String.fromCharCode( event.keyCode );
```

Comment:
I don't think this really matters in this case but when `event.keyCode >= 96 && event.keyCode <= 105` we get a number here and when it's less we get a string. Do we want to force character to always be a string for consistency?